### PR TITLE
Fixed where the friends are not sending get friends

### PR DESCRIPTION
**Pull request by sohank1**

### CODE REVIEW

**Comment by stuyy** [comment]
Master branch just got merged, please resolve conflicts

**Comment by sohank1** [comment]
Is this good

**Comment by sohank1** [comment]
the POST /post was broken because it uses that function. I imported that function into app.js and it should work now.

File: app.js [hunk 1]
```diff
@@ -40,12 +44,12 @@ app.post("/posts", async (req, res) => {
   await post.save();
 
   res.send(post);
-})
+});
 
 app.get("/friends/:username", async (req, res) => {
   const { username } = req.params;
   const user = await User.find({ username: username });
```
Comment (fahimmehraj) [inline app.js]: instead of find use findOne so that way it only returns one document, and then remove the array indexing that you have next to user

Comment (sohank1) [reply app.js]: fixed

**Pull request merged**